google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/facet/impl/AppEngineJavaeeSupportContributor.java

```diff
@@ -48,7 +48,9 @@ public void setupFrameworkSupport(JavaeeFrameworkSupportContributionModel model)
       return;
     }
 
-    setWebXmlServletVersion(model.getModule());
+    if (!appEngineStandardFacet.isJava8Runtime()) {
```

Comment:
technically it would be nice to check if its a java 7 runtime (think about java 9 in the future).

Reply:
lgtm otherwise

Reply:
Yeah I thought about this, but I found a ton of hidden assumptions elsewhere regarding the exclusive Java7/Java8 setup that won't exist in the future, and I figured it would be best to tackle Java9 separately. See usages of `AppEngineStandardFacet.isJava8Runtime()`, for example.

Reply:
what do you mean? like that for Java 7 the runtime is just "java"?